cli/export/register.ts

```
@@ -9,9 +11,21 @@ export const registerExport = (program: Command) => {
     .option("-f, --format <format>", "Output format")
     .option("-o, --output <path>", "Output file path")
     .action(async (file, options) => {
+      const format = options.format ?? "json"
+
+      if (format === "spice") {
+        const { circuitJson } = await generateCircuitJson({ filePath: file })
+        if (circuitJson) {
+          const spiceNetlist = circuitJsonToSpice(circuitJson as any)
+          const spiceString = spiceNetlist.toSpiceString()
+          console.log(spiceString)
```

nice, but isn't this supposed to output to a file? I can't remember exactly. Also why are you checking for the existence of circuitJson then silently failing? If the command fails we need to be loud about it